lib/outputTemplate.mustache

```diff
@@ -1,18 +1 @@
-<div class="reactPatternContainer" id="reactPatternContainer--{{{patternPartial}}}">
-  {{{htmlOutput}}}
-</div>
-
-<!-- pattern JSON (React props) -->
-<script data-type="patternJSON" id="patternJSON--{{{patternPartial}}}" type="application/json">
-  {{{json}}}
-</script>
-
-<!-- runtime React output -->
-<script>
-  {{{runtimeCode}}};
-
-  <!-- runtime rendering -->
-  var component = unknown.default;
-  var patternJSON = document.getElementById('patternJSON--{{{patternPartial}}}').textContent;
-  ReactDOM.render(React.createElement(component, JSON.parse(patternJSON)), document.getElementById('reactPatternContainer--{{{patternPartial}}}'));
-</script>
```

Comment:
This looks like the client-side stuff you mentioned you took out. This needs to be there for people who are developing (and documenting) interactive components. If it doesn't hurt your use-case, let's restore this. If it does, let's fix it.